cdap-gateway/src/test/java/co/cask/cdap/gateway/router/NettyRouterPipelineTest.java

```diff
@@ -221,7 +221,7 @@ private void deploy(int num) throws Exception {
       urlConn.setDoOutput(true);
       urlConn.setDoInput(true);
 
-      ByteStreams.copy(Locations.newInputSupplier(programJar), urlConn.getOutputStream());
+      ByteStreams.copy(programJar::getInputStream, urlConn.getOutputStream());
```

Comment:
Need to close the stream

Reply:
I believe that the method I’m calling will close it internally after
calling the getInputStream method.